README.md

```diff
@@ -4,24 +4,26 @@
 Minio Client (mc) provides a modern alternative to UNIX commands like ls, cat, cp, mirror, diff, find etc. It supports filesystems and Amazon S3 compatible cloud storage service (AWS Signature v2 and v4).
 
 ```
-ls       List files and folders.
-mb       Make a bucket or a folder.
-cat      Display file and object contents.
-pipe     Redirect STDIN to an object or file or STDOUT.
-share    Generate URL for sharing.
-cp       Copy files and objects.
-mirror   Mirror buckets and folders.
-find     Finds files which match the given set of parameters.
-stat     Stat contents of objects and folders.
-diff     List objects with size difference or missing between two folders or buckets.
-rm       Remove files and objects.
-events   Manage object notifications.
-watch    Watch for file and object events.
-policy   Manage anonymous access to objects.
-session  Manage saved sessions for cp command.
-config   Manage mc configuration file.
-update   Check for a new software update.
-version  Print version info.
+ls       list buckets and objects
+mb       make a bucket
+cat      display object contents
+pipe     stream STDIN to an object
+share    generate URL for temporary access to an object
+cp       copy objects
+mirror   synchronize objects to a remote site
+find     search for objects
+sql      run sql queries on objects
+stat     stat contents of objects
+diff     list differences in object name, size, and date between buckets
```

Comment:
Maybe ...
`list differences in name, size and date of objects in two buckets`

Reply:
Let's keep this for now, - this PR is nothing changing anything in the code. Send a PR to fix this yourself @ebozduman